### PR TITLE
Lecture 9

### PR DESCRIPTION
**Pull request by dc-mak**
Andrej for the content, Shaun to check the Makefile and splitting is ok.

### CODE REVIEW

File: tex/lecture9.tex [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+natural isomorphism $\lscat C (\phold, A) \natarr{\varphi} \lscat C (\phold,
+B)$ (which amounts to showing $ \Efrac{X \to A}{X \to B} $ `is natural in $X$',
+that is, holds for all $X \in \lscat C^\op $) and then define $f$ to be
+$\widehat{\varphi}$.
+
+\section{A product construction}
+
+Fix a shape: $ A \overset{\alpha}{\longleftarrow} P \morpharr{\beta} B $. 
+Define the functor $ K = \lscat C (\phold, A) \times \lscat C
+(\phold, B)$ (the product of two functors is a a functor) and the infranatural
```
Comment (Dreian) [inline tex/lecture9.tex:217]: This is the first occurrence of the word 'infranatural' -- should define it

File: tex/lecture9.tex [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+transformation $\varphi = \lscat C (\phold, P) \natarrow K $. Hence, for each $X
+\in \lscat C^\op $, we have: 
+
+\begin{center}
+\begin{tikzcd}
+    \lscat C(X, P) \arrow[rr] & & \lscat C (X, A) \times \lscat C (X, B) \\
```
Comment (Dreian) [inline tex/lecture9.tex]: Maybe put (\alpha \comp \phold, \beta \comp \phold)_X on this arrow?

File: tex/lecture9.tex [hunk 3]
```diff
@@ -0,0 +1,295 @@
...
+\end{tikzcd}
+\end{center}
+
+We now claim that $\varphi = (\alpha \circ \phold, \beta \circ \phold) = \setof{
+{(\alpha \circ \phold, \beta \circ \phold)}_X }_{X \in \lscat C^\op}$ satisifies
+the naturality condition.
+
+% Dhruv: The diagram presented in lectures did not include the C(X,P)/C(Y,P) row
+% or the C(-,P) arrows. It also said (alpha o g, beta o g), which I believe was
+% incorrect, so wrote it as followed.
```
Comment (Dreian) [inline tex/lecture9.tex]: I think (a o g, b o g) is the correct thing

File: tex/lecture9.tex [hunk 4]
```diff
@@ -0,0 +1,295 @@
...
+\end{tikzcd}
+\begin{tikzcd}
+              & id_P \arrow[rr, mapsto] && (P \morpharr{\alpha} A,\quad P \morpharr{\beta} B)
+\end{tikzcd}
+\end{center}
+
+\begin{exercise}
+
+% Dhruv: The original phrasing was something along the lines of: "Can we
+% reconstruct phi from A <-(a)- P -(b)-> B but I was unsure what that meant.
```
Comment (Dreian) [inline tex/lecture9.tex]: So phi = {phi_X}_X is the family of natural transformations C(-,P) => C(-,A) x C(-,B) that we want to characterise exactly, and I believe the idea is that we want to know whether we can reconstruct _all_ of them from just one?

File: tex/lecture9.tex [hunk 5]
```diff
@@ -0,0 +1,295 @@
...
+Prove $\varphi$ is natural. Finish the diagram such that it commutes, that is,
+define $\lscat C (g,P)$ and $K(g)$ for $g : Y \to X$ in terms of $\alpha$ and
+$\beta$ alone.
+\end{exercise}
+
+Having shown $\varphi$ is natural, we can then ask when $\varphi$ is a natural
+isomorphism.
+
+% Dhruv: There was an "exercise" here but I'm unsure what's left to do after
+% the following was presented in the lecture.
```
Comment (Dreian) [inline tex/lecture9.tex]: I don't think there was an exercise

**Review by Dreian — APPROVED** [review]
Minor suggestions

File: Makefile [hunk 6]
```diff
@@ -1,13 +1,28 @@
...
+BUILD := build
+SRC :=  lecture1.tex \
+	lecture2.tex \
+	lecture3.tex \
+	lecture4.tex \
+	lecture5.tex \
+	lecture6.tex \
+	lecture7.tex \
+	lecture8.tex \
+	lecture9.tex
```
Comment (technicalguy) [inline Makefile:13]: This is fine, but I would much prefer it if file splitting was in a separate pull request to the content from lecture 9.

Comment (dc-mak) [reply Makefile:13]: Yeah fair enough, that's on me, it's just lecture 9 had already started, the compile times were horrendous and I didn't want to perform git sorcery on a Sunday morning.

Comment (technicalguy) [reply Makefile:13]: No worries. How long does it take you to compile? It hardly takes me a second...

**Pull request merged**